packages/iTwinUI-react/src/core/Slider/Track.tsx

```diff
@@ -46,14 +46,15 @@ export type TrackProps = {
   sliderMin: number;
   sliderMax: number;
   values: number[];
+  orientation: Orientation; // TODO: Confirm if need documentation for this line
```

Comment:
I noticed that these variables did not have any documentation. Variables in `Slider.tsx` had documentation. So I just wanted to confirm whether not having any documentation here was alright or not.

Reply:
Since this component is only for internal use and props are kind of obvious, you don't need to always add comments.

Reply:
**[DONE]**